app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Kindly do the same thing you have done on line 2 for the associations to work.